data/asset_giveaway_1/getAssets.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import fs from 'fs';
+import {BigNumber} from 'ethers';
+import MerkleTree from '../../lib/merkleTree';
+import helpers from '../../lib/merkleTreeHelper';
+
+const {createDataArrayAssets, saltAssets} = helpers;
+
+const errors = false;
+
+function exitIfError() {
+  if (errors) {
+    process.exit(1);
+  }
+}
+
+type Assets = {
```

Comment:
I think Assets name is too generic
maybe something like `type AssetClaims = Claim[];`

of we coul simply have `type AssetClaims = {...}[];`